extensions/interactions/CodeRepl/CodeRepl.py

```diff
@@ -23,8 +23,11 @@ class CodeRepl(base.BaseInteraction):
     name = 'Code Editor'
     description = 'Allows learners to enter code and get it evaluated.'
     display_mode = base.DISPLAY_MODE_SUPPLEMENTAL
-    is_trainable = False
-    _dependency_ids = ['skulpt', 'codemirror']
+    is_trainable = True
+    _dependency_ids = [
+        'skulpt', 'codemirror', 'code_repl_prediction',
+        'winnowing_preprocessing', 'svm_predictionService',
```

Comment:
Shouldn't we get rid of some of these extra deps? We should have a backend test that iterates along every interaction and checks that its dependencies are valid, too.